.github/workflows/backport.yml

```diff
@@ -0,0 +1,19 @@
+name: Backport
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    runs-on: ubuntu-18.04
+    name: Backport
+    steps:
+      - name: Backport Bot
+        uses: Gaurav0/backport@v1.0.24
+        with:
+          bot_username: NordicBuilder
+          bot_token: 151a9b45052f9ee8be5a59963d31ad7b92c3ecb5
+          bot_token_key: 67bb1f1f998d546859786a4088917c65415c0ebd
```

Comment:
This isn't a secret, right?

Reply:
Good question, but it is safe backport-bot token only has read access.  
normally the tokens are private, but the backport-bot instructions explain how this is handled for public repositories.  
see step 6.  https://github.com/marketplace/actions/backport-bot

Reply:
Thanks for the pointer. So I guess `bot_token ^ bot_token_key` is the real token, which only has read access?

Either way, I'll take your word for it those instructions have been followed correctly, which seem safe enough to me.

Note that we can't merge this until https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/255 is in, at which point it will need to be rebased.

Reply:
bot_token ^ bot_token_key is correct.    
I will wait for #255